SimCalorimetry/EcalEBTrigPrimAlgos/interface/EcalEBFenixLinearizer.h

```diff
@@ -59,7 +59,7 @@ void EcalFenixLinearizer::process(const T &df, std::vector<int> &output_percry)
   // 1  8  11 18 21
   // 0  9  10 19 20
 
-  //std::cout << " EcalFenixLinearizer::process(const  .. DataFrame " << std::endl;
+  //std::cout << " EcalEBFenixLinearizer::process(const  .. DataFrame " << std::endl;
   for (int i = 0; i < df.size(); i++) {
```

Comment:
Why do you need to keep this loop on, it seems it does nothing here.

Reply:
Tbh I did not look at the code functionality itself. I just made the minimal changes to fix the issue. I can remove the loop if needed.

Reply:
Hi @srimanob I have pushed a new commit that removes the two useless loops. Could you stop the ongoing tests and start new ones please?

Reply:
Thanks.

Reply:
Thanks. Could you also launch the tests for #33349 in combination with this PR if you are on it?